projects/demo/src/modules/customization/icon-set/examples/1/index.html

```diff
@@ -1,7 +1,7 @@
 <div class="b-form">
     <tui-notification>
         As usual with the DI this is hierarchical. Meaning you can provide different
-        <code>TUI_ICONS_PATH</code>
```

Comment:
Let's change it in Stackblitz Starter too

https://github.com/Tinkoff/taiga-ui/blob/a0093a010a9263f2880c5e10fafd3df7c76c657a/projects/demo/src/modules/app/stackblitz/project-files/src/app/app.module.ts.md?plain=1#L18-L22